README.md

```diff
@@ -1,3 +1,53 @@
 # Coding Challenge: Github Api - React
 
-A minimalist react website to track a Github user's repos. 
\ No newline at end of file
+A minimalist react website to track a Github user's repos. 
+
+# Description & Requirements
+- [x] Your app should make use of React  
+- [x] Your repo tracker should have an input for users to give their GitHub username  
+- [x] After submitting their username, use the GitHub API to retrieve that user's list of repos  
+- [x] When selecting a repo, a User should be shown some data about that repo eg. issue count, stargazers, forks etc.  
+
+# Installation & usage
+
+### For our app to be run on your local machine:
+
+Clone this repo and navigate to the root directory of this repo.
+
+Get authentication from Github:   
+- Visit https://github.com/settings/tokens  
+- Generate New Token   
+- Paste your new token into `src/actions/index.js`  
+  `var options = {headers: {"Authorization": "token <<your token here>>"}};`
```

Comment:
Interesting that you used authorization here.  A nice touch!  Curious to know what would happen without this token though?  Would I not be able to run `npm run dev`? (Not read the code base yet, so maybe some extra explanation here might be useful).